cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/oauth/OAuthProvider.java

```diff
@@ -87,7 +87,6 @@ public OAuthProvider build() {
       Preconditions.checkNotNull(name, "OAuth provider name missing");
       Preconditions.checkNotNull(loginURL, "Login URL missing");
       Preconditions.checkNotNull(tokenRefreshURL, "Token refresh URL missing");
-      Preconditions.checkNotNull(clientCreds, "OAuth client credentials missing");
```

Comment:
Please mark field / getter as nullable

Reply:
Done

Reply:
Great, let me know when pushed this change. I can see all other changes

Reply:
Done.

Missed pushing the file earlier.